### PR TITLE
Docstrings for strava

### PR DESCRIPTION
**Pull request by phillprice**
This might help docstrings?

### CODE REVIEW

File: homeassistant/components/strava/__init__.py [hunk 1]
```diff
@@ -47,8 +55,17 @@
 
 
 class StravaData:
-    """A model which stores the Strava data."""
-
+    """
```
Comment (stv0g) [inline homeassistant/components/strava/__init__.py]: This is a single-line docstring. There is no need to span it over 3 lines.

File: homeassistant/components/strava/__init__.py [hunk 2]
```diff
@@ -47,8 +55,17 @@
...
+    A model which stores the Strava data.
+    """
+
+    ##
```
Comment (stv0g) [inline homeassistant/components/strava/__init__.py]: We dont need two docstrings here. Please stick the format described in PEP-257

File: homeassistant/components/strava/__init__.py [hunk 3]
```diff
@@ -69,12 +86,24 @@ def __init__(self, hass, config):
 
     @property
     def is_authorized(self):
-        """Check if there is a (possiblly expired) OAuth2 token."""
+        """
+        Check if there is a (possiblly expired) OAuth2 token.
+        
+        @param      self  The object
+
```
Comment (stv0g) [inline homeassistant/components/strava/__init__.py]: Dont add empty lines @param, @return, etc

File: homeassistant/components/strava/__init__.py [hunk 4]
```diff
@@ -35,6 +35,14 @@
 }, extra=vol.ALLOW_EXTRA)
 
 
+##
+## @brief      { function_description }
```
Comment (stv0g) [inline homeassistant/components/strava/__init__.py]: Please use a meaning full description. We can't leave these placeholders in here.

File: homeassistant/components/strava/__init__.py [hunk 5]
```diff
@@ -47,8 +55,17 @@
...
-    """A model which stores the Strava data."""
-
+    """
+    A model which stores the Strava data.
+    """
+
+    ##
+    ## @brief      Constructs the object.
+    ##
+    ## @param      self    The object
```
Comment (stv0g) [inline homeassistant/components/strava/__init__.py]: Home Assistant seems to use the Sphinx doctoring format:
https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html

See here for [an example](https://github.com/home-assistant/home-assistant/blob/b3e60df82aec2116040a2814cd3c9d09aef320ed/homeassistant/components/hangouts/hangups_utils.py#L16)

File: homeassistant/components/strava/__init__.py [hunk 6]
```diff
@@ -35,6 +35,14 @@
 }, extra=vol.ALLOW_EXTRA)
 
 
+##
+## @brief      { function_description }
+##
+## @param      hass    The hass
```
Comment (stv0g) [inline homeassistant/components/strava/__init__.py]: In my opinion, I would not document parameters such as `self` or `hass` as they are self explanatory. "The object" is not helpful and just clutters the code.. I think we can stick mostly with single line docstrings.

**Comment by phillprice** [comment]
FYI This is autogenerated through doxygen ;) 

will fixup where I can :)

**Comment by stv0g** [comment]
Hi @phillprice,

I added the missing docstrings myself.

**Pull request closed without merging**